code/src/item00.c

```diff
@@ -16,7 +16,30 @@
 
 void EnItem00_rInit(Actor* thisx, GlobalContext* globalCtx) {
     EnItem00* item = THIS;
-
+    s16 item00Type = item->actor.params & 0x00FF;
+    //If no ammo drops is chosen as an option, overrides the incoming ammo or magic drop with a blue rupee
+    if(gSettingsContext.ammoDrops == AMMODROPS_NONE){		
+        if (item00Type == 0x04 ||
+        item00Type == 0x05 ||
+        item00Type == 0x08 ||
+        item00Type == 0x09 ||
+        item00Type == 0x0A ||
+        item00Type == 0x0B ||
+        item00Type == 0x0C ||
+        item00Type == 0x0E ||
+        item00Type == 0x0F ||
+        item00Type == 0x10 ||
+        item00Type == 0x12 ||
+        item00Type == 0x19) {
```

Comment:
I think it would be nice to use 
[this enum](https://github.com/zeldaret/oot/blob/3fbdccbdbac4113ae04d77216b2a28b405e07734/include/z64actor.h#L243) for this, it could probably just be put in item00.h
https://github.com/zeldaret/oot/blob/3fbdccbdbac4113ae04d77216b2a28b405e07734/include/z64actor.h#L243
It also might look better as a switch block like
```
switch(item00Type) {
    case ITEM00_BOMBS_A:
    case ITEM00_ARROWS_SINGLE:
    case ITEM00_ARROWS_SMALL:
        // ...
    case ITEM00_BOMBS_SPECIAL:
        item->actor.params = (item->actor.params & 0xFF00) | 0x01;

}
```